app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AccountPage.scala.html

```diff
@@ -41,9 +41,11 @@ <h1 class="govuk-heading-xl">@messages("account-homepage.title", undertaking.nam
     content = Text(messages("account-homepage.warning", "[TODO: NAME]"))
   ))
 
-  @govukInsetText(InsetText(
-    content = Text(messages("account-homepage.inset", "[TODO: DATE]"))
-  ))
+    @if(isTimeToReport.equals(true)) {
```

Comment:
`if(isTimeToReport)` should do the trick fine

Reply:
o yes  🤦‍♀️